pytorch_blade/src/torch-mlir/include/torch-mlir/Conversion/TorchToMhlo/MhloLegalizeUtils.h

```diff
@@ -124,6 +126,48 @@ llvm::Optional<Value> getMhloShapeOfTensor(
     Value value);
 
 Value getNumelOfTensor(PatternRewriter& rewriter, Operation* op, Value value);
+
+llvm::Optional<Value> getUnsqueezedTensor(
```

Comment:
nit: `create` seems better than `get`

Reply:
It's named with "get" just to be consistent with other functions.